### PR TITLE
doc(install): update ITEE installation page

### PR DESCRIPTION
**Pull request by tuntoja**
## Description

Update ITEE installation documentation.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: versioned_docs/version-24.04/administration/custom-login.md [hunk 1]
```diff
@@ -10,46 +10,22 @@ You can customize some elements of your Centreon platform:
...
-
-   - **License Manager** (if not yet installed)
-   - **IT Edition Extensions**
+> To purchase one, contact [Centreon](mailto:sales@centreon.com).
+
+## Configure the extension
+
+1. Go to **Administration > Customize Centreon**. The different options are:
+   
+   - **Platform name**: define a name for your platform (e.g. Test**, **Production). This name will be visible both on the login page (next to the logo), and in the banner once you are logged in.
```
Comment (tuntoja) [inline versioned_docs/version-24.04/administration/custom-login.md:22]: strange stars are not going well ***

File: i18n/fr/docusaurus-plugin-content-docs/version-24.04/administration/custom-login.md [hunk 2]
```diff
@@ -10,45 +10,22 @@ Vous pouvez personnaliser certains éléments de votre plateforme Centreon :
 - **Page de connexion :** vous pouvez changer l'arrière-plan et le logo, ainsi qu'ajouter un texte.
 - **Nom de la plateforme :** vous pouvez définir un nom pour votre plateforme (par exemple Test, Production). Celui-ci sera affiché dans le bandeau lorsque vous serez connecté.
 
-Pour pouvoir personnaliser votre plateforme Centreon, vous devez installer l'extension **Centreon IT Edition Extensions**.
+Pour pouvoir personnaliser votre plateforme Centreon, vous devez utiliser l'extension **Centreon IT Edition Extensions**. Celle-ci est installée par défaut sur votre plateforme.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-24.04/administration/custom-login.md:13]:
```suggestion
La personnalisation de votre plateforme Centreon utilise l'extension **Centreon IT Edition Extensions**. Celle-ci est installée par défaut sur votre plateforme.
```

File: versioned_docs/version-24.04/administration/custom-login.md [hunk 3]
```diff
@@ -10,46 +10,22 @@ You can customize some elements of your Centreon platform:
 - **Login page:** you can change the background and the logo, and add a text.
 - **Name of the platform:** you can define a name for your platform (e.g. Test, Production) that will be displayed in the banner once you are logged in.
 
-To be able to customize your Centreon platform, you need to install the **Centreon IT Edition Extensions** extension.
+To be able to customize your Centreon platform, you need to use the **Centreon IT Edition Extensions** extension. This extension is automatically installed on your platform.
```
Comment (cg-tw) [inline versioned_docs/version-24.04/administration/custom-login.md:13]:
```suggestion
Customizing your Centreon platform required the **Centreon IT Edition Extensions** extension. This extension is automatically installed on your platform.
```

File: versioned_docs/version-24.04/administration/custom-login.md (hunk 1 above)
Comment (cg-tw) [inline versioned_docs/version-24.04/administration/custom-login.md:22]:
```suggestion
   - **Platform name**: define a name for your platform (e.g. **Test**, **Production**). This name will be visible both on the login page (next to the logo), and in the banner once you are logged in.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — DISMISSED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by cg-tw — DISMISSED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**